### PR TITLE
djview4 4.10

### PR DESCRIPTION
**Pull request by theirix**
Nnew version of djview4. Build process changes a little in upstream and fixed in the formula.

Upgraded djview4 to 4.10
Dropped unneeded "myswap" patch.
Added patches to fix configure.ac and Makefile.am

### CODE REVIEW

File: Library/Formula/djview4.rb [hunk 1]
```diff
@@ -2,22 +2,27 @@
...
   depends_on 'djvulibre'
   depends_on 'qt'
 
-  # Patch for Qt 4.8 compatibility. See:
-  # https://build.opensuse.org/package/view_file?file=djview4-qt-4.8.patch&package=djvulibre-djview4&project=graphics&rev=8c40ae0f91469bb1af80f36c24516251
-  # When updating this formula, check if this patch is still needed.
+  # Patch build files for correct flag and dependency handling (known problems at upstream)
   patch :DATA
 
+  ENV['NOCONFIGURE'] = '1'
```
Comment (jacknagel) [inline Library/Formula/djview4.rb]: this should be inside the install method.

File: Library/Formula/djview4.rb [hunk 2]
```diff
@@ -2,22 +2,28 @@
...
+  depends_on 'autoconf' => :build
+  depends_on 'automake' => :build
+  depends_on 'libtool' => :build
   depends_on 'djvulibre'
   depends_on 'qt'
 
-  # Patch for Qt 4.8 compatibility. See:
-  # https://build.opensuse.org/package/view_file?file=djview4-qt-4.8.patch&package=djvulibre-djview4&project=graphics&rev=8c40ae0f91469bb1af80f36c24516251
-  # When updating this formula, check if this patch is still needed.
+  # Patch build files for correct flag and dependency handling (known problems at upstream)
```
Comment (xu-cheng) [inline Library/Formula/djview4.rb]: Can you link the upstream issue url in the comment? Thanks.

**Comment by theirix** [comment]
These problems are fixed in development version of djview. 
First chunk is mentioned at commit https://sourceforge.net/p/djvu/djview-git/ci/f1c0fad16267f4eaf34a3fcc8692e1a280755cdc/ and  issue https://sourceforge.net/p/djvu/patches/34/
Second chunk fixes a bug referenced by upstream commit https://sourceforge.net/p/djvu/djview-git/ci/a66dc1178255a95298979677f424d1c9fd793d15/ and does not have matching issue.

I hope djview maintainers will release soon next patch version with these changes.

**Comment by theirix** [comment]
I updated the pull request to upstream version 4.10.3 that includes two initial patches.

**Pull request closed without merging**